plugins/org.eclipse.papyrus.uml.diagram.sequence.runtime/src/org/eclipse/papyrus/uml/diagram/sequence/runtime/internal/preferences/LightweightSequenceDiagramPreferencePage.java

```diff
@@ -33,4 +43,26 @@ public LightweightSequenceDiagramPreferencePage() {
 		setPreferenceKey(ViewTypes.LIGHTWEIGHT_SEQUENCE_DIAGRAM);
 	}
 
+	@Override
+	protected Control createContents(Composite parent) {
+		Group optionsGroup = new Group(parent, 2);
+		optionsGroup.setLayout(new GridLayout());
+		optionsGroup.setText(Messages.SynchronousMessagePreferencesLabel);
+
+		autoCreateExecution = new BooleanFieldEditor(
+				LightweightSequenceDiagramPreferences.AUTO_CREATE_EXEC_AFTER_SYNC_MESSAGE,
+				Messages.CreateExecutionAfterSyncMessageAutomaticallyLabel, optionsGroup);
+		autoCreateExecution.setPage(this);
+
+		autoCreateReply = new BooleanFieldEditor(
```

Comment:
Enablement of this preference should be conditional on the execution specification preference being selected. And it should be indented to visually indicate that dependency.